### DISCUSSION TITLE
Outcome and Potential  capabilities of this course.

### DISCUSSION THREAD

**Discussion by vigneshk004** [Cyfrin/foundry-full-course-cu#2742] [Q&A] [answered]
Would a person in general be able to work on his/her own cryptocurrency once after completing this course

**Comment by EngrPips** [comment]
I believe so.

**Reply by vigneshk004** [reply]
Thank you for your time.

**Reply by EngrPips** [reply]
Anytime

**Comment by BowtiedHarpyEagle** [comment, marked as answer]
I think you can. Since one of the projects in the Advanced Foundry course is a stable coin project.

**Reply by vigneshk004** [reply]
Thank you for your time
